src/background.js

```diff
@@ -1774,6 +1794,7 @@ function init() {
 								return false;
 							});
 					}
+					ghosteryDebug.addAccountEvent('app started', 'not signed in');
```

Comment:
Does this need to be wrapped in an `else`?

Reply:
you shouldn't have to as the function returns in the preceding lines.